### ISSUE TITLE
DropDownList keyboard navigation does not work in IE.

### ISSUE THREAD

**Issue by nemalp** [telerik/kendo-ui-core#884]
The widget cannot be focused when using Access key + w. 

Can be reproduced here: http://demos.telerik.com/kendo-ui/dropdownlist/keyboard-navigation
Steps: 
1. Open the demo in IE. 
2. Hold Access key + w to focus the widget. 
3. The DropDownList is not focused. 

Reported in TicketID 946366

**Comment by ggkrustev** [comment]
It seems that the widget does not copy the accesskey attribute to the wrapper. Hence the widget cannot be focused by "Access key" + letter combination.
